script.js

```diff
@@ -131,18 +131,17 @@ class Game{
 
     calculate_card_total(hand){
         this.score = 0;
```

Comment:
does `this.score` need to be attached to the game class?

Reply:
Updated.